ReactiveCocoa/ControlSubject.swift

```diff
@@ -0,0 +1,44 @@
+import ReactiveSwift
+import enum Result.NoError
+
+/// Represents a subject of an interactive control.
+///
+/// Generally speaking, `ControlSubject` emits changes only for the user
+/// interactions in which the subject specializes.
+public final class ControlSubject<Value> {
```

Comment:
@mdiep I agree regarding the name, but disagree about "softening" the protocol behavior so we don't have to come up with a new thing. :)

Consider for example the `Sequence` vs `Collection` in the standard library. Imagine if `Sequence`s never existed, and "some `Collection`s just return -1 for `count` when they are infinite in size.